.github/ISSUE_TEMPLATE/improvement.md

```diff
@@ -0,0 +1,17 @@
+---
+name: ✅ Improvement suggestion
+about: Suggest an improvement for the project
+title: '[IMPROVEMENT]'
+labels: 'Type/Improvement'
+assignees: ''
+
+---
+
+**Is your suggestion related to a experience ? Please describe.**
```

Comment:
a experience ? --> an experience?

Reply:
Fixed https://github.com/wso2/product-is/pull/8055/commits/1165aa490b71d547eeb8a5d263ff5453d64bf03d